conf/sample/sidecar-injector/webhook-create-signed-cert.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```

Comment:
You may want to put a copyright at the top of this file with appropriate information. This also looks close to the one found here [1], so if that's the case, we should ensure to attribute it correctly. And finally, we've merged shellcheck support, so after you rebase, you may hit some static analysis errors in this file, which you'll need to cleanup.

Reply:
Yes, these scripts are coming from here - https://github.com/morvencao/kube-mutating-webhook-tutorial/tree/master/deployment
What should the license include in this case??